src/components/App.jsx

```diff
@@ -4,7 +4,7 @@ import cloneDeep from 'lodash.clonedeep'
 import clamp from 'lodash.clamp'
 import get from 'lodash.get'
 import {unset} from 'lodash'
-import {arrayMove} from 'react-sortable-hoc'
+import arrayMove from 'array-move'
```

Comment:
`{arrayMove}` will not be exported in the newer versions of `react-sortable-hoc`, move to `array-move` instead.